### PR TITLE
Implement Bookstore structure with react components and routes

### PR DESCRIPTION
**Pull request by abdallahmalima**
## In this part I:
- Add project structure with  JSX
- Using Components
- Add Routes for pages

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import NavBar from './components/NavBar';
+import BookPage from './Pages/BookPage';
+import CategoryPage from './Pages/CategoryPage';
+
+const App = () => (
+  <div className="App">
+     <NavBar/>
+     <Routes>
+       <Route path="/" element={<BookPage/>}/>
+       <Route path="/books" element={<BookPage/>}/>
```
Comment (ha-manel) [inline src/App.js]: - [ ] Kindly remove this route, one route `/` which is the default route is enough for the book page 💯

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @abdallahmalima 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Professional README file ✔️

## Required Changes:

- [ ] Kindly add the MIT file, the link in your README file is broken 👍 

_Kindly check the comments below_


------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -278,7 +211,7 @@ I would like to thank...
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](https://github.com/abdallahmalima/Bookstore/blob/dev/MIT.md) licensed.
```
Comment (danmainah) [inline README.md:214]: - [ ] @abdallahmalima great work in implementing the changes requested, kindly customize the [MIT](https://github.com/abdallahmalima/Bookstore/blob/dev/MIT.md) license with your info and the app info.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @abdallahmalima , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️ No linter errors
✔️ Github flow was correctly used


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @abdallahmalima 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ No linter errors. 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**